weave/trace/weave_client.py

```diff
@@ -534,6 +534,13 @@ class ApplyScorerSuccess:
             wc._send_score_call(self, score_call, scorer_ref_uri)
         return apply_scorer_result
 
+    def to_dict(self) -> dict:
```

Comment:
I would more concretely type this so that the dict coming out has strongly typed fields instead of relying on the implicit fields from the dataclass